### PR TITLE
FIX: Duplicated metadata in product is not allowed

### PR DESCRIPTION
**Pull request by jmmovilla**
@jframos @hmunfru

### CODE REVIEW

**Comment by jframos** [comment]
LGTM

File: rest-api/src/main/java/com/telefonica/euro_iaas/sdc/rest/validation/ProductResourceValidatorImpl.java [hunk 1]
```diff
@@ -266,6 +272,20 @@ private void checkDependence(String dependence) throws InvalidProductException {
 
     }
 
+    private boolean areMetadataDuplicated (List<Metadata> metadatas) {
```
Comment (flopezag) [inline rest-api/src/main/java/com/telefonica/euro_iaas/sdc/rest/validation/ProductResourceValidatorImpl.java]: Different comments here:
1) Where are you initialised the support_metadata list? You create it but you do not assign values?
2) You are taking the metadata with index 0 and compare with the metadata 0, for sure duplicated...

File: rest-api/src/test/java/com/telefonica/euro_iaas/sdc/rest/validation/ProductResourceValidatorImplTest.java [hunk 2]
```diff
@@ -471,4 +471,16 @@ public void testValidateAttributesBadType5() throws InvalidEntityException, Alre
 
         verify(systemPropertiesProvider, times(1)).getProperty(SystemPropertiesProvider.AVAILABLE_ATTRIBUTE_TYPES);
     }
+    @Test(expected = InvalidEntityException.class)
+    public void testValidateMetadataDuplicated() throws Exception {
```
Comment (flopezag) [inline rest-api/src/test/java/com/telefonica/euro_iaas/sdc/rest/validation/ProductResourceValidatorImplTest.java]: You have to add a new test in which the metadata are not duplicated...

**Comment by flopezag** [comment]
LGTM

**Pull request merged**